src/gui/properties/peerlistwidget.cpp

```diff
@@ -51,464 +51,456 @@
+        }
+        else {
+            // Add new peer
+            m_peerItems[peer_ip] = addPeer(peer_ip, torrent, peer);
+            m_peerAddresses[peer_ip] = addr;
+            // Resolve peer host name is asked
+            if (m_resolver)
+                m_resolver->resolve(peer_ip);
+        }
+    }
+    // Delete peers that are gone
+    QSetIterator<QString> it(old_peers_set);
+    while (it.hasNext()) {
+        const QString& ip = it.next();
+        m_missingFlags.remove(ip);
+        m_peerAddresses.remove(ip);
+        QStandardItem *item = m_peerItems.take(ip);
+        m_listModel->removeRow(item->row());
     }
-  }
-  // Delete peers that are gone
-  QSetIterator<QString> it(old_peers_set);
-  while(it.hasNext()) {
-    const QString& ip = it.next();
-    m_missingFlags.remove(ip);
-    m_peerAddresses.remove(ip);
-    QStandardItem *item = m_peerItems.take(ip);
-    m_listModel->removeRow(item->row());
-  }
 }
 
-QStandardItem* PeerListWidget::addPeer(const QString& ip, BitTorrent::TorrentHandle *const torrent, const BitTorrent::PeerInfo &peer) {
-  int row = m_listModel->rowCount();
-  // Adding Peer to peer list
-  m_listModel->insertRow(row);
-  m_listModel->setData(m_listModel->index(row, PeerListDelegate::IP), ip);
-  m_listModel->setData(m_listModel->index(row, PeerListDelegate::IP), ip, Qt::ToolTipRole);
-  m_listModel->setData(m_listModel->index(row, PeerListDelegate::PORT), peer.address().port);
-  m_listModel->setData(m_listModel->index(row, PeerListDelegate::IP_HIDDEN), ip);
-  if (m_displayFlags) {
-    const QIcon ico = GuiIconProvider::instance()->getFlagIcon(peer.country());
-    if (!ico.isNull()) {
-      m_listModel->setData(m_listModel->index(row, PeerListDelegate::COUNTRY), ico, Qt::DecorationRole);
-      const QString country_name = Net::GeoIPManager::CountryName(peer.country());
-      m_listModel->setData(m_listModel->index(row, PeerListDelegate::COUNTRY), country_name, Qt::ToolTipRole);
-    } else {
-      m_missingFlags.insert(ip);
+QStandardItem* PeerListWidget::addPeer(const QString& ip, BitTorrent::TorrentHandle *const torrent, const BitTorrent::PeerInfo &peer)
+{
+    int row = m_listModel->rowCount();
+    // Adding Peer to peer list
+    m_listModel->insertRow(row);
+    m_listModel->setData(m_listModel->index(row, PeerListDelegate::IP), ip);
+    m_listModel->setData(m_listModel->index(row, PeerListDelegate::IP), ip, Qt::ToolTipRole);
+    m_listModel->setData(m_listModel->index(row, PeerListDelegate::PORT), peer.address().port);
+    m_listModel->setData(m_listModel->index(row, PeerListDelegate::IP_HIDDEN), ip);
+    if (m_displayFlags) {
+        const QIcon ico = GuiIconProvider::instance()->getFlagIcon(peer.country());
+        if (!ico.isNull()) {
+            m_listModel->setData(m_listModel->index(row, PeerListDelegate::COUNTRY), ico, Qt::DecorationRole);
+            const QString country_name = Net::GeoIPManager::CountryName(peer.country());
```

Comment:
You can move ```Net::GeoIPManager::CountryName(peer.country())``` directly in ```m_listModel->setData()``` call (without creating variable).
Or rename country_name => countryName